extension/src/util/array.ts

```diff
@@ -19,7 +19,7 @@ export const joinTruthyItems = (array: (string | undefined)[], sep = ' ') =>
 export const sameContents = (
   array: (null | string | number | undefined)[],
   otherArray: (null | string | number | undefined)[]
-) => isEqual(array.sort(), otherArray.sort())
+) => isEqual([...array].sort(), [...otherArray].sort())
```

Comment:
[F] This was an interesting bug to nail down... by interesting I mean confusing and frustrating.

Reply:
I'll add `etc/no-assign-mutated-array` once this is merged.